### PR TITLE
Updating clipboard copy to include inline variants

### LINKED ISSUE #2409: Add guidance on using new "inline" copy to clipboard variant
We are adding a new lightweight inline variant of the Clipboard Copy component. Want to add guidance on when/why to use vs. the standard/existing variant.

cc @mmenestr

### PR DESCRIPTION
**Pull request by lwrigh**
Updating existing design documentation to include inline clipboard copy design documentation.

closes #2409

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md [hunk 1]
```diff
@@ -38,6 +37,15 @@ An expandable clipboard copy allows for long lines of text to be stored in an ex
 
 <img src="./img/expandable.png" alt="Example of an expandable clipboard copy component" width="455"/>
 
+**Inline**
+
+An inline clipboard copy is placed directly inline with other text or used on it's own when you want a lighter weight clipboard copy treatment.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:42]: Should be its not it's!

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md [hunk 2]
```diff
@@ -38,6 +37,15 @@ An expandable clipboard copy allows for long lines of text to be stored in an ex
...
+
+<img src="./img/inline-one-action.png" alt="Example of an inline clipboard copy component with one action" width="355"/>
+
+An inline clipboard copy can include one or two actions buttons next to it. The second action is customizable and optional.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:46]: should be "two action buttons", not "two actionS buttons"

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md [hunk 3]
```diff
@@ -38,6 +37,15 @@ An expandable clipboard copy allows for long lines of text to be stored in an ex
 
 <img src="./img/expandable.png" alt="Example of an expandable clipboard copy component" width="455"/>
 
+**Inline**
+
+An inline clipboard copy is placed directly inline with other text or used on it's own when you want a lighter weight clipboard copy treatment.
+
+<img src="./img/inline-one-action.png" alt="Example of an inline clipboard copy component with one action" width="355"/>
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:44]: Last comment is that these images are a little blurry! Make sure to export the art board at x2 PNG, but keep the width set to the original sketch art board width!

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md (hunk 1 above)
Comment (lwrigh) [reply packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:42]: Thanks for catching that, I'll update it.

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md (hunk 2 above)
Comment (lwrigh) [reply packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:46]: Same here, thanks for catching that!

File: packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md (hunk 3 above)
Comment (lwrigh) [reply packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md:44]: I can do that, thank you for letting me know. I'll re-export the images.

**Comment by lwrigh** [comment]
> This looks good @lwrigh . There is one thing that came up yesterday that I should make you aware of. React already was using "inline" to refer to the standard variant. We can't change that without breaking existing users, so we decided to call the new variant "inline compact." Not sure if that should change any naming in this doc.

@mcarrano Thank you for bringing that up! I think it makes sense to update the naming here to 'inline compact' so it's more aligned with the React variant. I can update the markdown file to reflect naming update.

**Review by mcarrano — APPROVED** [review]
Looks great. Thanks @lwrigh !

**Comment by lwrigh** [comment]
Thank you no problem!

**Review by mmenestr — APPROVED** [review]
Looks great Laura, thank you!

**Pull request merged**